src/wallet/wallet2.cpp

```diff
@@ -4226,19 +4092,19 @@ void wallet2::get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>>
       std::unordered_set<uint64_t> seen_indices;
       // request more for rct in base recent (locked) coinbases are picked, since they're locked for longer
       size_t requested_outputs_count = base_requested_outputs_count + (td.is_rct() ? CRYPTONOTE_MINED_MONEY_UNLOCK_WINDOW - CRYPTONOTE_DEFAULT_TX_SPENDABLE_AGE : 0);
-      size_t start = req.outputs.size();
+      size_t start = req_outputs.size();
 
       // if there are just enough outputs to mix with, use all of them.
       // Eventually this should become impossible.
       uint64_t num_outs = 0, num_recent_outs = 0;
-      for (auto he: resp_t.result.histogram)
+      for (auto& he: histogram)
       {
         if (he.amount == amount)
         {
-          LOG_PRINT_L2("Found " << print_money(amount) << ": " << he.total_instances << " total, "
-              << he.unlocked_instances << " unlocked, " << he.recent_instances << " recent");
-          num_outs = he.unlocked_instances;
-          num_recent_outs = he.recent_instances;
+          LOG_PRINT_L2("Found " << print_money(amount) << ": " << he.total_count << " total, "
+              << he.unlocked_count << " unlocked, " << he.recent_count << " recent");
+          num_outs = he.unlocked_count;
+          num_recent_outs = he.recent_count;
```

Comment:
Why more gratuitous changes :/

Reply:
when I read "unlocked_instances" I don't know at a glance that it means a count of the unlocked instances rather than the unlocked instances themselves.  Using the word "count" instead cleanly conveys this information.